### DISCUSSION TITLE
Delta 2 Max

### DISCUSSION THREAD

**Discussion by svenerbe** [vwt12eh8/hassio-ecoflow#91] [General]
Is there any plan to support delta 2 max? i try to connect the a delta 2 max but no answer from the device so far.

**Comment by digitalurban** [comment]
Just also adding this, it would be great to see the delta 2 max added if thats possible :)

**Comment by luxmoel** [comment]
I would also be very happy about an integration!!!

**Comment by Neewieedo** [comment]
Hi, is it possible to connect the Delta 2 or Delta 2 Max with the actual version?
